data_steward/tools/retract_data_gcs.py

```diff
@@ -151,40 +168,73 @@ def get_response():
 
 
 def get_integer(num_str):
-    try:
-        num = int(ast.literal_eval(str(num_str)))
-        if isinstance(num, int):
-            return num
-    except (SyntaxError, TypeError, ValueError):
-        return None
+    """
+    Converts an integer in string form to integer form
+    Throws SyntaxError/TypeError/ValueError if input string is not an integer and terminates
 
+    :param num_str: an integer in string form
+    :return: integer form of num_str
+    """
+    num = int(ast.literal_eval(str(num_str)))
```

Comment:
`int(num_str)` should suffice, no? Please see the test I added.

Reply:
I think there were other cases covered by `ast.literal_eval`, but I cannot recollect them now. I think `int(num_str)` should suffice, looking at the test cases.